### ISSUE TITLE
[Errno 2] No such file or directory: './annotations/gencode.v43lift37.annotation.txt.gz'

### ISSUE THREAD

**Issue by Manuel-DominguezCBG** [broadinstitute/SpliceAI-lookup#38]
This is the full output I got

The first lines looks ok for me but there is an File not found error at the end

`
++ NUM_THREADS=1
++ HOST=127.0.0.1
++ PORT=8080
++ TIMEOUT=1800
++ gunicorn -w 1 -t 1800 -b 127.0.0.1:8080 server:app
[2023-05-24 10:01:20 +0100] [155600] [INFO] Starting gunicorn 20.1.0
[2023-05-24 10:01:20 +0100] [155600] [INFO] Listening at: http://127.0.0.1:8080 (155600)
[2023-05-24 10:01:20 +0100] [155600] [INFO] Using worker: sync
[2023-05-24 10:01:20 +0100] [155604] [INFO] Booting worker with pid: 155604
Using TensorFlow backend.
/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:542: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:544: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:545: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
/home/mdb1c20/SpliceAI-lookup/server.py:34: FutureWarning: Passing a negative integer is deprecated in version 1.0 and will not be supported in future version. Instead, use None to not limit the column width.
  pd.set_option('max_colwidth', -1)
Reading rate limit outlier IPs: /mainfs/home/mdb1c20/SpliceAI-lookup/rate_limit_outlier_ips.txt
Current list of rate limit outlier IPs: []
Loading ./annotations/gencode.v43lift37.annotation.txt.gz
[2023-05-24 10:02:22 +0100] [155604] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/home/mdb1c20/.conda/envs/lookup/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/mdb1c20/SpliceAI-lookup/server.py", line 127, in <module>
    df = pd.read_table(annotation_path, dtype={"TX_START": int, "TX_END": int})
  File "/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/pandas/io/parsers.py", line 767, in read_table
    return read_csv(**locals())
  File "/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/pandas/io/parsers.py", line 688, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/pandas/io/parsers.py", line 454, in _read
    parser = TextFileReader(fp_or_buf, **kwds)
  File "/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/pandas/io/parsers.py", line 948, in __init__
    self._make_engine(self.engine)
  File "/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/pandas/io/parsers.py", line 1180, in _make_engine
    self._engine = CParserWrapper(self.f, **self.options)
  File "/home/mdb1c20/.conda/envs/lookup/lib/python3.6/site-packages/pandas/io/parsers.py", line 2010, in __init__
    self._reader = parsers.TextReader(src, **kwds)
  File "pandas/_libs/parsers.pyx", line 382, in pandas._libs.parsers.TextReader.__cinit__
  File "pandas/_libs/parsers.pyx", line 620, in pandas._libs.parsers.TextReader._setup_parser_source
  File "/home/mdb1c20/.conda/envs/lookup/lib/python3.6/gzip.py", line 163, in __init__
    fileobj = self.myfileobj = builtins.open(filename, mode or 'rb')
FileNotFoundError: [Errno 2] No such file or directory: './annotations/gencode.v43lift37.annotation.txt.gz'
[2023-05-24 10:02:23 +0100] [155604] [INFO] Worker exiting (pid: 155604)
[2023-05-24 10:02:23 +0100] [155600] [INFO] Shutting down: Master
[2023-05-24 10:02:23 +0100] [155600] [INFO] Reason: Worker failed to boot.
`

I am going to add steps I needed to do to reach this step

I followed the instructions in the README file but I have found many packages were not load, starting with Pangolin, gffutils biopython, pandas, pyfastx and torch. I needed to do this the previous time I wanted to install this in my Mac and now in the Linux too.

Thank you

**Comment by Manuel-DominguezCBG** [comment]
I have tried to find that file on my own and I have found this https://www.gencodegenes.org/human/release_43lift37.html. However, I believe you have modified the file because although the file name is the same the format in your code is .txt. and comparing the one I have downloaded gencode.v43lift37.annotation.gtf.gz with a similar one the repo provides gencode.v38lift37.annotation.txt.gz, the structure of the data is different. Or maybe it is a different file with the same name I am not able to find.
